vrx_gazebo/src/gymkhana_scoring_plugin.cc

```diff
@@ -123,9 +123,12 @@ void GymkhanaScoringPlugin::ChannelCallback(
     // Determine whether channel has been crossed before timeout
     if (!this->channelCrossed)
     {
-      if (msg->state == "finished" && !msg->timed_out)
```

Comment:
Removing this condition causes the task to exit immediately when a gate is crossed incorrectly. The  `navigation_scoring_plugin` reports a finished state, and the score is equal to the timeout score, so it finishes. Is this intended? I'm pointing this out because it seems like your testing instructions suggest that I should be able to make at least two illegal moves.